cmd/kube-proxy/app/server_test.go

```diff
@@ -283,8 +288,8 @@ nodePortAddresses:
 
 		assert.NoError(t, err, "unexpected error for %s: %v", tc.name, err)
 
-		if !reflect.DeepEqual(expected, config) {
-			t.Fatalf("unexpected config for %s, diff = %s", tc.name, cmp.Diff(config, expected))
+		if !reflect.DeepEqual(expected.Logging, config.Logging) && cmp.Diff(config.Logging, expected.Logging) != "" {
+			t.Fatalf("unexpected config for %s, diff = %s", tc.name, cmp.Diff(config.Logging, expected.Logging))
```

Comment:
This seems to be removing the check that the rest of `config` is correct

Reply:
oh thanks, it's my test code here